src/app/containers/ConsentBanner/index.stories.jsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { RequestContextProvider } from '../../contexts/RequestContext';
+import { ServiceContextProvider } from '../../contexts/ServiceContext';
+import ConsentBanner from '.';
+import AmpDecorator from '../../../../.storybook/helpers/ampDecorator';
+
+const getConsentBanner = platform => (
+  <ServiceContextProvider service="news">
+    <RequestContextProvider
+      platform={platform}
+      isUK
+      origin="https://www.bbc.co.uk"
+      pageType="article"
+      id="c0000000000o"
+      statsDestination="NEWS_PS_TEST"
+      statsPageIdentifier="news.articles.c0000000000o"
+    >
+      <ConsentBanner />
+      This container uses cookies to conditionally render. If you can not see
```

Comment:
Nice additional text 😎

Reply:
Props to Drew for the support
![](https://media.giphy.com/media/3o7TKuuvlqt1wXGxGM/giphy.gif)